src/screens/TopicPageScreen/index.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import _ from 'lodash';
 import SimpleToast from 'react-native-simple-toast';
-import {Animated, Platform, StyleSheet, View} from 'react-native';
+import {Animated, Platform, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `StyleSheet` from the import statement suggests that it's no longer being used in this file. This is a good practice as it helps to reduce unnecessary imports and keeps the code clean.
```diff
- import {Animated, Platform, StyleSheet, View} from 'react-native';
+ import {Animated, Platform, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->